specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -637,6 +637,13 @@
               "$ref": "#/definitions/ConstrainedResource"
             }
           },
+          {
+            "name": "force",
+            "in": "query",
```

Comment:
This is anti-pattern. Query parameters should be used in PUT requests.

Reply:
This API is following an existing Azure Synapse API that uses the same pattern:
https://docs.microsoft.com/en-us/rest/api/synapse/bigdatapools/createorupdate 

Calling PUT on this ConstrainedResource will effectively create a synapse spark pool by passing the request to Synapse.

So matching the existing API was the main reasoning here.
This is the same for the 201 accepted below. Synapse use this pattern and we align with it so we can return the same response.